Source/Classes/AnimatedImages/PINAnimatedImageView.h

```diff
@@ -23,4 +23,9 @@
 @property (nullable, nonatomic, strong) NSString *animatedImageRunLoopMode;
 @property (nonatomic, assign, getter=isPlaybackPaused) BOOL playbackPaused;
 
+/**
+ @abstract Seek to a play head position of an animated image.
```

Comment:
Can we add to the comment and mention that this may not be performant? Because the GIF is currently memory backed and caches ahead to optimize for performance.